Source/Core/Clients/OpenIdClient.swift

```
@@ -30,8 +30,11 @@ internal class OpenIdClient: HTTPClient {
     }
 	
  	func performLogout() -> Observable<Void> {
+		guard let refreshToken = AirMap.authService.refreshToken,
+			  let clientId = AirMapConfiguration.custom?.clientId
+		else { return Observable.error(AirMapError.unauthorized) }
```

Let's fail silently here instead of returning an error.

---

Fixed